arduino/smartcar/smartcar.ino

```diff
@@ -77,31 +75,53 @@ void setup() {
  
   mqtt.subscribe("/smartcar/control/#", 1);
   mqtt.onMessage([](String topic, String message) {
-    if (topic == "/smartcar/control/throttle") {
-      car.setSpeed(message.toInt());
-    } else if (topic == "/smartcar/control/steering") {
-      car.setAngle(message.toInt());
+    if (topic == "/smartcar/control/takeInput") {
+      takeInput(message);
     } else {
       Serial.println(topic + " " + message);
     }
   });
 }
  
-void obstacle() {
-    const auto distance = front.getDistance();
- if (distance > 0 && distance < 100) {
-    car.setSpeed(0);
- }
-}
- 
-void carGo() {
-      car.setSpeed(60);
-}
- 
+void takeInput(String input) {
+        int inputSelection = input.substring(0).toInt();
+            int throttle;
+            int angle;
+            if(input.length() > 1) {
+              unsigned int stringInput = input.substring(1).toInt(); 
+              throttle = stringInput;
+              angle = stringInput;
```

Comment:
What is the point of using two variable that are set to the same value?

Reply:
The review has not been approved yet, only the sketch has been tested in SMCE. So still in progress of reviewing but aye there's some code clean up and amendments to be made

Reply:
You are correct since the input being recieved will be the same. Didnt realise. I will update and do a new commit. Thanks 👍